packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/add/add.controller.js

```diff
@@ -6,47 +9,196 @@ export default class {
   }
 
   $onInit() {
-    this.trackDashboard(`service_integration::add_${this.engineName}`, 'page');
+    this.trackDashboard(
+      `service_integration::add_${this.database.engine}`,
+      'page',
+    );
+    this.showNoServiceMessage = false;
+    this.parameters = {};
+    this.stringParameterOptions = STRING_PARAMETER_OPTIONS;
+
+    // aggregate capabilities so we only have one of each type, with an array as source and destination
+    // engines
+    this.aggregatedIntegrationCapabilities = this.integrationCapabilities.reduce(
+      (acc, curr) => {
+        const existingCapability = acc.find(
+          (capability) => capability.type === curr.type,
+        );
+        if (existingCapability) {
+          if (!existingCapability.sourceEngines.includes(curr.sourceEngine)) {
+            existingCapability.sourceEngines.push(curr.sourceEngine);
+          }
+          if (
+            !existingCapability.destinationEngines.includes(
+              curr.destinationEngine,
+            )
+          ) {
+            existingCapability.destinationEngines.push(curr.destinationEngine);
+          }
+        } else {
+          const capabilityArray = {
+            type: curr.type,
+            parameters: curr.parameters,
+            sourceEngines: [curr.sourceEngine],
+            destinationEngines: [curr.destinationEngine],
+          };
+          acc.push(capabilityArray);
+        }
+        return acc;
+      },
+      [],
+    );
```

Comment:
Same as https://github.com/ovh/manager/pull/7186#discussion_r943360296

Reply:
Same answer here, the linter won't allow it.